.github/workflows/release.yml

```diff
@@ -185,7 +185,7 @@ jobs:
           JIRA_DESCRIPTION: ${{ env.jira_description }}
```

Comment:
The code change seems to reorder the variables in the `RELEASE_NOTES` string. It moves `${RELEASE_NOTES}` to a different position in the string. This might change the output format of the `RELEASE_NOTES`.